account_payment_order_validation_move_line/wizard/account_payment_line_create.py

```diff
@@ -0,0 +1,24 @@
+# © 2015-2016 Akretion (<https://www.akretion.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class AccountPaymentLineCreate(models.TransientModel):
+    _inherit = "account.payment.line.create"
+
+    select_for_payment_move_line = fields.Boolean(
```

Comment:
Rename to `selected_for_payment_filter` for example (not sure that move_line is really relevant here)